addons/hr_contract/views/hr_contract_views.xml

```diff
@@ -19,15 +19,15 @@
                     <xpath expr="//field[@name='resource_calendar_id']" position="replace">
                         <field name="calendar_mismatch" invisible="1"/>
                         <label for="resource_calendar_id"/>
-                        <div>
-                            <field name="resource_calendar_id" required="1" nolabel="1"/>
-                            <span attrs="{'invisible': [('calendar_mismatch', '=', False)]}"
-                                class="fa fa-exclamation-triangle text-danger o_calendar_warning ms-3"
-                                data-tooltip-template="hr_contract.CalendarMismatch"
-                                data-tooltip-info='{
-                                        "text":"Calendar Mismatch : The employee&apos;s calendar does not match this contract&apos;s calendar. This could lead to unexpected behaviors."
-                                    }'>
-                            </span>
+                        <div class="row">
+                            <div class="col-10">
+                                <field name="resource_calendar_id" required="1" nolabel="1"/>
+                            </div>
+                            <div class="col-2 pt-1">
+                                <widget
+                                    name="contract_warning_toolip"
+                                    attrs="{'invisible': [('calendar_mismatch', '=', False)]}"/>
+                            </div>
```

Comment:
why complexifying the arch? I'm not sure you're supposed to play with bootstrap cols
cc @adr-odoo